playbooks/fedora-30/pre-common.yaml

```diff
@@ -0,0 +1,13 @@
+- name: Load registry image f30/fedora-toolbox:30 in pre-phase and auto restart when registry is flaky
```

Comment:
Nitpick: just `Pull registry.fedoraproject.org/f30/fedora-toolbox:30` might be better when looking at the test logs. The logs already list this under `pre`, so the details about the phase might not be necessary.

Reply:
Tweaked.